src/applications/personalization/profile/tests/e2e/direct-deposit/update-flow.cypress.spec.js

```diff
@@ -173,19 +175,19 @@ describe('Direct Deposit', () => {
         // register and the bank info form does not open
         force: true,
       });
-      fillInBankInfoForm('CNP');
-      fillInBankInfoForm('EDU');
-      saveNewBankInfo('CNP');
-      saveNewBankInfo('EDU');
-      // This scan will be run while the bank info is saving and the
-      // LoadingButton is in its "loading" state. This would throw an aXe error
-      // if LoadingButton.loadingText was not set
-      cy.axeCheck();
-      // Now wait for the update API calls to resolve to failures...
-      cy.findAllByText(/we couldn’t update your bank info/i).should(
-        'have.length',
-        '2',
-      );
+      // fillInBankInfoForm('CNP');
+      // fillInBankInfoForm('EDU');
+      // saveNewBankInfo('CNP');
+      // saveNewBankInfo('EDU');
+      // // This scan will be run while the bank info is saving and the
+      // // LoadingButton is in its "loading" state. This would throw an aXe error
+      // // if LoadingButton.loadingText was not set
+      // cy.axeCheck();
+      // // Now wait for the update API calls to resolve to failures...
+      // cy.findAllByText(/we couldn’t update your bank info/i).should(
+      //   'have.length',
+      //   '2',
+      // );
```

Comment:
Should we plan on removing these completely at a later data?

Reply:
Ooops, That should be commented back in. Thanks for the catch!